example/properties_directory/structural/covalent_bonds.json

```diff
@@ -0,0 +1,36 @@
+[
+    {
+        "atomPair": [
+            {
+                "id": 1,
+                "value": "H"
+            },
+            {
+                "id": 2,
+                "value": "O"
+            }
+        ],
+        "bondType": "single",
+        "bondLength": {
+            "units": "angstrom",
+            "value": 0.99
+        }
+    },
```

Comment:
Also - on the above - it might be a good idea to include the bonds information into the basis